libs/ui-toolkit/src/components/toast/toast.css

```diff
@@ -0,0 +1,20 @@
+.toast-initial {
+  top: 20px;
+  opacity: 0;
+  max-height: 0;
+  border: 0;
+  margin-bottom: 0;
+}
+
+.toast-showing {
+  opacity: 1;
+  transition: all 0.3s;
```

Comment:
fix (blocking): There are some scroll bars popping in and out as the toast animates off screen can we fix that.

TBH I'd prefer to leave the animation out for now as Joelle would like to set out some animation guidelines for us as part of the design system.

![Uploading Nov-09-2022 12-03-20.gif…]()